packages/terra-site/src/examples/dialog/DialogDefault.jsx

```diff
@@ -1,17 +1,23 @@
+/* eslint-disable no-alert */
 import React from 'react';
 import Dialog from 'terra-dialog';
 
-const header = <span>Header Stuff</span>;
+const header = 'Header Stuff';
```

Comment:
If it's possible, it would be better if we allowed this to take in react nodes as well. With react nodes, you can build components that easily let you opt in and out of overflowing text

Reply:
Can't. This gets passed into ActionHeader, which requires String.